data/cities.yml

```diff
@@ -1489,11 +1489,11 @@
 - geo:
     lat: 35.7795897
     lng: -78.6381787
-  name: Raleigh, NC, US
+  name: Raleigh, NC, USA
```

Comment:
Most events in the states contain "USA" in their names, so I renamed the rest accordingly.